src/stores/Frame/FrameStore.ts

```diff
@@ -2739,6 +2875,7 @@ export class FrameStore {
 
         // udpate center position for setting inputs
         this.center = this.spatialTransform.transformCoordinate(this.spatialReference.center, false);
+        this.setOffsetCenter(this.center.x, this.center.y);
```

Comment:
Shall we update the offset center only when coord offset is enabled?

Reply:
yes, it is better.